prime-router/metadata/organizations.yml

```diff
@@ -16,21 +16,23 @@
       - name: elr
         topic: covid-19
         schema: az/az-covid-19
-        jurisdictionalFilter: { standard.patient_state: AZ }
+        jurisdictionalFilter: { standard.testing_lab_state: AZ }
```

Comment:
Now filtering on the Lab's location, not the patient's.    Will need further tweaking.